src/app/CommandHandlerImpl.h

```diff
@@ -436,7 +436,7 @@ class CommandHandlerImpl : public CommandHandler
      */
     bool IsGroupRequest() { return mGroupRequest; }
 
-    bool ResponsesAccepted() { return !(mGroupRequest || mpResponder == nullptr); }
```

Comment:
This change was unrelated, but made it more readable to reason about when responses are accepted (i.e. mResponder must not be null), vs the previous conditional which negated the whole thing